diff --git a/base/BUILD.gn b/base/BUILD.gn
index 070547ddf67a01f5310893ed569263da2ce8042d..3acc689bbebf97458af196897cae2e943f5aa648 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4240,6 +4240,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
