diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 37b87c2328ab4ef23176a5bab01ea0b52cde35cb..9873c15d0a363f7d8c16c98412e06a12c9f4c3fd 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1666,6 +1666,7 @@ config("chromium_code") {
       ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
