diff --git a/chrome/browser/resources/tools/optimize_webui.gni b/chrome/browser/resources/tools/optimize_webui.gni
index 25732ec00e575af482bfe95cd479633679eb0b16..33eb8312bd8c5c772107677b9143fe4998eba1a0 100644
--- a/chrome/browser/resources/tools/optimize_webui.gni
+++ b/chrome/browser/resources/tools/optimize_webui.gni
@@ -70,6 +70,8 @@ template("optimize_webui") {
       "//resources/|$resources_path",
     ]
 
+    import("//brave/browser/resources/tools/brave_optimize_webui.gni") deps += brave_webui_deps external_paths += brave_webui_external_paths
+
     if (defined(invoker.external_paths)) {
       external_paths += invoker.external_paths
     }
