diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index e44d8f08c6bc09c985e09bafb8adb8f73b5b5e7b..a383157d4bede31a593ae0d199e26adf4051df09 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -317,6 +317,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (enable_pdf) {
@@ -5034,7 +5035,7 @@ config("disable_thinlto_cache_flags") {
 }
 
 config("pdb_larger_than_4gb") {
-  if (is_win && symbol_level == 2 && use_thin_lto) {
+  if (is_win) {
     # These binaries create PDBs larger than 4 GiB. Increasing the PDB page
     # size allows larger PDBs, but not all tools can handle such large PDBs
     # yet.
