diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 677e26dc4a77c0e54b558c414914111becca5281..2d5d0ebacb5a52202e6a0951b380b06ed1850661 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3200,6 +3200,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
