diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index e5a1e952cc763af5e489c6eebf14ec4a29e7b904..0692b6769c0436cc896fcc456dfa52eb391bd52a 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -337,6 +337,7 @@ std::string GetAndroidOSInfo(
 
   // Send information about the device.
   bool semicolon_inserted = false;
+  BRAVE_GET_ANDROID_OS_INFO
   if (include_android_model == IncludeAndroidModel::Include) {
     std::string android_device_name = BuildModelInfo();
     if (!android_device_name.empty()) {
