diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 9465ee1bda8dce382275f9acd709a29ceb088002..44c1b1a7bf4d04de3eb4111845d058204f67f86e 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6045,7 +6045,6 @@ def CheckPythonShebang(input_api, output_api):
                 file))
     return result
 
-
 def CheckBatchAnnotation(input_api, output_api):
     """Checks that tests have either @Batch or @DoNotBatch annotation. If this
     is not an instrumentation test, disregard."""
@@ -6108,3 +6107,5 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+
+if not globals().get('__name__'): from lib.import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
