diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 0c1c89027890545df05357e9964137696ca0adbb..91ddc57b35b8ac2f9363e7a695254a6f1c7a8739 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6027,7 +6027,6 @@ def CheckPythonShebang(input_api, output_api):
                 file))
     return result
 
-
 def CheckBatchAnnotation(input_api, output_api):
     """Checks that tests have either @Batch or @DoNotBatch annotation. If this
     is not an instrumentation test, disregard."""
@@ -6090,3 +6089,5 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+
+if not globals().get('__name__'): from lib.import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
