diff --git a/third_party/blink/public/devtools_protocol/browser_protocol.pdl b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
index 873459a670782a5c24e7d6aafd4e0b2d06eb8558..c7992576250fd62a11ca18bcc2f5d9ee3ab30eb1 100644
--- a/third_party/blink/public/devtools_protocol/browser_protocol.pdl
+++ b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
@@ -7125,6 +7125,8 @@ domain Page
       web-share
       window-placement
       xr-spatial-tracking
+      ethereum
+      solana
 
   # Reason for a permissions policy feature to be disabled.
   experimental type PermissionsPolicyBlockReason extends string
